tscached/kquery.py

```diff
@@ -22,6 +24,16 @@ def from_request(cls, request, redis_client):
         for metric in request.get('metrics', []):
             new = cls(redis_client)
             new.query = metric
+
+            # This seemingly terrifying kludge makes it easy to merge chunks of MTS together.
+            # For all aggregators, we remove align_sampling and add align_start_time.
+            # Otherwise, partial windows (if aggregated) will appear on every seam in the data. Info:
+            # https://kairosdb.github.io/docs/build/html/restapi/QueryMetrics.html#metric-properties
+            for agg_ndx in xrange(len(new.query.get('aggregators', []))):
```

Comment:
Do we handle any aggregators in KQuery's currently? Is grafana (or whatever) able to request a data set with an aggregator set? 

Also, on a side note, would we want to implement aggregators for large (i.e. 3 hour) data sets as an effective form of downsampling to reduce the total data returned and maybe speed up query results? j/w